public/src/admin/manage/category.js

```diff
@@ -64,7 +64,7 @@ define('admin/manage/category', [
 		}
 
 		
-		$('form.category input, form.category select')
+		$('form.category').find('input,textarea,select')
```

Comment:
I don't bellieve we have a `textarea` in the admin `category.tpl`... is this necessary?